index.html

```diff
@@ -41,13 +54,14 @@
 
       <p>Previous Editions of Pycon India have been supported by companies including
       ZeOmega, Microsoft, Google, GoIbibo, Zopper, Red Hat etc. If you or your company 
-      would like to be a part of this elite list, feel free to drop a line on sponsorship_AT_in.pycon.org</p>
+      would like to be a part of this elite list, feel free to drop a line on <span class="blue"><u>sponsorship_AT_in.pycon.org </u></span>.</p>
 
 
-      <p>For any other clarification, please feel free to touch base with the organizing
-      team at <b>contact_AT_in.pycon.org</b></p>
+      <p>For any other clarification, please feel free to touch base with the organizing team at <span class="blue"><u>contact_AT_in.pycon.org</u></span>.
+       </p>
```

Comment:
sorry . really sorry because i know its a mistake but its my habit now . I am trying my best to improve it and keep trying to improve this . I know everything should be maintained properly in a orderly manner . REALLY SORRY . and thank you for pointing out my mistake. AND FROM NEXT TIME I WILL TRY MY BEST TO GIVE A PROPER ORDERLY MANNER IN CONVENTIONAL FORM .